maquetado/scss/details.scss

```diff
@@ -21,6 +27,7 @@
 
 .book-info {
   flex-direction: column;
+  width: 540px;
```

Comment:
Careful, if your width is higher than 250px, then you should do: 

```
width: 100%; 
max-width: 540px;
```

For responsive reasons.